hack/go-lint.sh

```diff
@@ -20,6 +20,6 @@ fi
 rm -f golint_errors
 # The command in or will fetch the latest tag available for golangci-lint and install in $GOPATH/bin/
 which golangci-lint > /dev/null || curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
-golangci-lint run ./... --enable golint
+golangci-lint run ./... --enable golint --timeout 2m0s
```

Comment:
this seems to be a really short timeout or ?

Reply:
By default it was 1m0s, I think 2m0s is enough

Reply:
oki